src/cljs/main/broadfcui/common/links.cljs

```diff
@@ -13,22 +13,22 @@
    contents])
 
 (defn create-download
-  ([label url]
-   (create-download label url nil)) ; nil :download is fine with React
-  ([label url filename]
-   [:a {:href url
-        :download filename}
-    label
-    icons/download-icon]))
+  ([label url] (create-download label url nil false)) ; nil :download is fine with React
+  ([label url filename] (create-download label url filename false))
+  ([label url filename create-internal?]
+   (let [attributes {:href url :download filename} ]
+     (if create-internal?
+       (create-internal attributes label)
+       [:a attributes label icons/download-icon]))))
 
```

Comment:
This is so we can use `create-internal` for the Download link so that it's visually the same as the Populate Json link